src/main/resources/db/migration/0300/V320__ProjectOverallScores.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE accelerator.project_overall_scores (
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    overall_score FLOAT,
+    summary TEXT,
+    details_url TEXT,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    PRIMARY KEY (project_id)
```

Comment:
If you want, you can put this on line 2, `project_id BIGINT PRIMARY KEY REFERENCES projects ON DELETE CASCADE`. This is fine too.